scripts/ci/copyright_check.py

```diff
@@ -28,6 +28,17 @@ def get_header(has_shebang, regexes):
     indices = range(init_idx, init_idx + len(regexes))
     return zip(regexes, indices)
 
+
+def match_somewhere(regexes, lines):
+    maybe_match_head_index = [index for index, line in enumerate(lines) if regexes[0].search(line)][:1]
+    if maybe_match_head_index and maybe_match_head_index[0] + len(regexes) <= len(lines):
+        match_head_index = maybe_match_head_index[0]
+        matches = [regex.search(lines[match_head_index + index]) for index, regex in enumerate(regexes)]
+        return all(matches)
+    else:
+        print('error return')
```

Comment:
This shouldn't be printed.

Reply:
Thanks. Forgot to get rid of it after debugging.

Reply:
Fixed with 4c48d334d4fd1631529108a23ee127754bab87be